gensim/corpora/wikicorpus.py

```diff
@@ -214,9 +214,10 @@ def extract_pages(f, filter_namespaces=False):
             title = elem.find(title_path).text
             text = elem.find(text_path).text
 
-            ns = elem.find(ns_path).text
```

Comment:
When I transform old enwiki dump (2010 in this notebook), Attribution Error was returned as follows.

`  File "***/lib/python3.5/site-packages/gensim/corpora/wikicorpus.py", line 211, in extract_pages
    ns = elem.find(ns_path).text
AttributeError: 'NoneType' object has no attribute 'text'`

To avoid this error, I fixed wikicorpus.py.